aws/aws.go

```diff
@@ -44,13 +44,6 @@ type AwsAccount struct {
 	ParentId       sql.NullInt64 `json:"-"`
 }
 
-const (
-	// assumeRoleDuration is the duration in seconds assumed-role
-	// credentials are requested to last. 3600 seconds is the maximum valid
-	// value.
-	assumeRoleDuration = 3600
```

Comment:
@Dayrion1 : why did you decide to remove it? what is the default value now?

Reply:
This is variable wasn't used anywhere in the code.